assignments/images-challenge/code.js

```diff
@@ -23,3 +23,16 @@
  *
  *  clear()
  */
+let urMom = 150
+let radius = urMom
+const lineOfCircles = (radius) => {
+  const diameter = 2*radius
+  const dist = Math.floor(width/diameter)*diameter
+  const extra = width-dist
+  let x = 0
+  while (x < width-extra){ 
+    drawFilledCircle(x+radius+extra/2, height/2, radius, 'red')
+    x += diameter
+  }
+}
+lineOfCircles (radius);
```

Comment:
And finally, you don't really need `urMom` or `radius` since the only place the global `radius` is used is right here so you could just change this to call the function with whatever value you want, like `150`.

```suggestion
lineOfCircles(150);
```